modules/rubiconAnalyticsAdapter.js

```diff
@@ -36,6 +37,13 @@ const cache = {
   timeouts: {},
 };
 
+let referrerHostname;
```

Comment:
Noticed `referrerHostname` is defined outside of where `referrer` is and `getHostNameFromReferer` will only be called once.

Is it possible for `referrer` to be different for different `sendMessage` calls?

Reply:
No this value only needs to be set once since the referrer is the url of the page prebid was loaded.

This should not change no matter how many auctions are run.

Unless I am missing something.

I just did not want to run the urlparse every time.

Reply:
I guess it's possible that a site would have a single application that changes content, and the url, without actually reloading.  but seems super unlikely that the hostname would ever change.

Reply:
@robertrmartinez That makes sense. Was just checking.